cypress/e2e/search.cy.js

```diff
@@ -0,0 +1,233 @@
+import SearchPageObjects from "../pageObjects/searchPage";
+
+const searchPage = new SearchPageObjects();
+const tags = ['@search', '@authentcation', '@common', '@root']
+const devices = ['ipad-2', 'ipad-mini', 'iphone-3', 'iphone-4', 'iphone-5', 'iphone-6', 'iphone-6+', 'iphone-7', 'iphone-8', 'iphone-xr', 'iphone-se2', 'macbook-11', 'macbook-13', 'macbook-15', 'macbook-16', 'samsung-note9', 'samsung-s10']
+const filterSearch = ['Last name A-Z', 'Last name Z-A', 'Best match']
+const numberOfResults = ['40', '20', '12']
+
+describe('Search Page',{tags: tags}, ()=> {
+    beforeEach(()=> {
+        cy.login()
+    })
+    it('should search for a property, person and tenure', {'tags': '@SmokeTest'}, ()=> {
+        searchPage.visit();
+        //property
+        const searchTerm = "Avenue"
+        searchPage.searchContainer().type(searchTerm);
+        searchPage.searchButton().click();
+
+        searchPage.searchResults().contains(searchTerm.replace(/\*/g, ""), { matchCase: false });
+        searchPage.searchResults().contains("Property type")
+        searchPage.searchResults().contains("Tenure")
+        searchPage.searchResults().contains("UPRN")
+
+        cy.get('[class*="govuk-back-link lbh-back-link"]').click();
+
+        //tenure
+        searchPage.tenureRadioButton().click();
+
+        const searchTermTenure = "Avenue"
+        searchPage.searchContainer().type(searchTermTenure);
+        searchPage.searchButton().click();
+
+        searchPage.searchResults().contains(searchTermTenure.replace(/\*/g, ""), { matchCase: false });
+        searchPage.searchResults().contains("Tenure payment ref");
+
+        cy.get('[class*="govuk-back-link lbh-back-link"]').click();
+
+        //person    
+        searchPage.personRadioButton().click();
+
+        const searchTermPerson = "Test"
+        searchPage.searchContainer().type(searchTermPerson);
+        searchPage.searchButton().click();
+
+        searchPage.searchResults().contains(searchTermPerson.replace(/\*/g, ""), { matchCase: false });
+    })
+
+    devices.forEach((device) => {
+        it('should search on all device types', () => {
+            searchPage.visit();
+            cy.viewport(`${device}`);
+            searchPage.personRadioButton().click();
+    
+            const searchTerm = "Test"
+            searchPage.searchContainer().type(searchTerm);
+            searchPage.searchButton().click();
+    
+            searchPage.searchResults().contains(searchTerm.replace(/\*/g, ""), { matchCase: false });
```

Comment:
Okay, I see it's all over the place. I will stop pointing these out.
Simple find and replace would fix it.
```js
searchTermTenure.replaceAll('*','')
```

Reply:
An alternative would be adding a method `.findSearchTerm(term)` to the SearchPage